tests/test_smoke.py

```diff
@@ -174,7 +180,7 @@ def run_one_test(test: Test) -> Tuple[int, str, str]:
             test.teardown,
             stdout=log_file,
             stderr=subprocess.STDOUT,
-            timeout=10 * 60,  # 10 mins
+            timeout=15 * 60,  # 15 mins
```

Comment:
It seems the 10 mins work for the original tests, why do we change it here?

Reply:
It is a remnant. Changed back now.